elastic/kibana
# [Security Solution] Update data field for JA3 fingerprint

## Summary

issue: https://github.com/elastic/kibana/issues/91803

Update `tls.server.ja3s` in the query to `tls.client.ja3s`:

Steps to verify:
1. Go to Network > TLS
2. Click on the inspect button of TLS table, and check the request
3. `tls.client.ja3` should be used for the `ja3` query

Update the query for TSL table:
`{
  "aggs": {
    "count": {
      "cardinality": {
        "field": "tls.server.hash.sha1"
      }
    },
    "sha1": {
      "terms": {
        "field": "tls.server.hash.sha1",
        "size": 10,
        "order": {
          "_key": "desc"
        }
      },
      "aggs": {
        "issuers": {
          "terms": {
            "field": "tls.server.issuer"
          }
        },
        "subjects": {
          "terms": {
            "field": "tls.server.subject"
          }
        },
        "not_after": {
          "terms": {
            "field": "tls.server.not_after"
          }
        },
        "ja3": {
          "terms": {
            "field": "tls.client.ja3s"
          }
        }
      }
    }
  },
  "query": {
    "bool": {
      "filter": [
        {
          "bool": {
            "must": [],
            "filter": [],
            "should": [],
            "must_not": []
          }
        },
        {
          "range": {
            "@timestamp": {
              "gte": "2021-09-19T23:00:00.000Z",
              "lte": "2021-09-20T22:59:59.999Z",
              "format": "strict_date_optional_time"
            }
          }
        }
      ]
    }
  },
  "size": 0
}`

### Checklist

Delete any items that are not applicable to this PR.
